sycl/source/detail/queue_impl.cpp

```diff
@@ -201,7 +201,7 @@ void *queue_impl::instrumentationProlog(const detail::code_location &CodeLoc,
   xpti::utils::StringHelper NG;
   Name = NG.nameWithAddress<queue_impl *>("queue.wait", this);
 
-  if (!CodeLoc.fileName()) {
+  if (CodeLoc.fileName()) {
```

Comment:
that is odd.   Nice catch!
@aelovikov-intel  - shouldn't that have been caught by a test?  If not, should we develop one?

Reply:
>  I probably need to verify it fails without this change...

In fact, I think the Jenkins/pre-ci-linux failure there shows that the test catches the issue against unmodified HEAD, so the verification could be assumed done.